### ISSUE TITLE
document have some mistake in article:Understanding the JavaScript API for Office

### ISSUE THREAD

**Issue by cherubic** [OfficeDev/office-js-docs-pr#21]
Article: [Understanding the JavaScript API for Office](https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/develop/understanding-the-javascript-api-for-office.md)

Issue: the code snippet in the The second level heading **Initialization reason** below the heading **Initializing your add-in** have some mistake,Seem to missing some symbols. 
Original:
```
Office.initialize = function (reason) {
    $(document).ready(function () {
      switch (reason) {
        case 'inserted': console.log('The add-in was just inserted.');
        case 'documentOpened': console.log('The add-in is already part of the document.');
    }
}
```
Should be:
```
Office.initialize = function (reason) {
    $(document).ready(function () {
        switch (reason) {
            case 'inserted': console.log('The add-in was just inserted.');
            case 'documentOpened': console.log('The add-in is already part of the document.');
        }
    });
 };
```

**Comment by Reezaali** [comment]
Hello @cherubic, thanks for reporting this. Would you like to make the fix to the documentation, and then submit a pull request to get the contribution on your profile? Thanks!

**Comment by kbrandl** [comment]
Fixed by [PR #26](https://github.com/OfficeDev/office-js-docs-pr/pull/26). This change will be reflected on the live site the next time **master** gets merged to **live** (most likely Tuesday 12/12).
